orchestra/static/orchestra/team-info/team-info-card.html

```diff
@@ -33,7 +33,17 @@ <h3>
           <tbody>
             <tr ng-repeat="assignment in teamInfoCard.assignments">
               <td>{{assignment.role}}</td>
-              <td>{{assignment.worker.username}}</td>
+              <td>{{assignment.worker.username}}
+                <button
```

Comment:
Only show this button to project admins (other parts of the team info card show how to do this)

Reply:
and don't show the button if the task is `Complete`.